### DISCUSSION TITLE
Improve handling of `This check was cancelled`

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#17010] [Actions]
https://github.com/jsoref/github-docs/runs/6529364826?check_suite_focus=true

![image](https://user-images.githubusercontent.com/2119212/169587237-57cb7548-f67e-4d04-9d86-9a5b563c586f.png)

> 🛑 
> This check was cancelled

There isn't much to go on here.

➡️  Ideally `cancelled` or the entire thing would somehow link one to _what cancelled it_.

https://github.com/jsoref/github-docs/actions/runs/2359826057

There's technically a _bit_ more here, but not much:

![image](https://user-images.githubusercontent.com/2119212/169587446-89a6937c-deb4-4cba-8f2f-c6aac68c8e9e.png)

Annotations
1 error and 1 warning
❌  [Close invalid Repo Sync PRs](https://github.com/jsoref/github-docs/runs/6529364826?check_suite_focus=true)
An error occurred while provisioning resources (Error Type: Disconnect).
⚠️  [Close invalid Repo Sync PRs](https://github.com/jsoref/github-docs/runs/6529364826?check_suite_focus=true)
Received request to deprovision: The request was cancelled by the remote provider.

---

https://github.community/t/jobs-on-macos-latest-sometimes-gets-cancelled-an-error-occurred-while-provisioning-resource/17434/15

---

Note: I don't understand why anything was provisioned, ideally this workflow should just perform some `if`s and decide "yeah, this workflow doesn't apply to this workflow".

**Comment by legraina** [comment]
Hi,

I have the same issue, please help !
I am trying to make a workflow work on a public repo (https://github.com/wssuite/coin/) and the test cannot pass.
However, I do not have any error in the log (when I'm able to see the log) and I only have this general error:
```
Received request to deprovision: The request was cancelled by the remote provider.
```
I have read that some unavailability of the servers may caused this issue, but I've tried several times at different periods, and I still have this issue.
Any idea?
